_posts/0000-01-02-nd2806.md

```diff
@@ -0,0 +1,5 @@
+--- 
+layout: slide title: 
+"Welcome to our second slide!" 
+--- 
+Your text Use the left arrow to go back!
```

Comment:
As Sylvia Plath said:


**"I have the choice of being constantly active and happy or introspectively passive and sad. Or I can go mad by ricocheting in between."**